src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -141,11 +141,12 @@ public void runRightUp() {
     }
 
     public boolean isLiftUp() {
-        return rightLiftEncoder.getPosition() < LiftConstants.maxExtension;
+        return rightLiftEncoder.getPosition() < LiftConstants.maxExtension 
```

Comment:
So this takes into account both sides for the command, however you also changed the up functions to use this condition so if the left side is all the way up but the right side isn’t, then the left will keep going up and cause damage to the robot. 
We need 2 is up functions, one for left and one for right. You could add a 3rd to combine the two if you want or just combine both in the is finished of the command